code/game/objects/items.dm

```diff
@@ -53,6 +52,10 @@
 	var/vending_cat = null// subcategory for vending machines.
 	var/list/dynamic_overlay[0] //For items which need to slightly alter their on-mob appearance while being worn.
 
+	var/shrapnel_amount = 0 // How many pieces of shrapnel it disintegrates into. This variable also decides it the item disintegrates or not!
```

Comment:
@PJB3005 Wouldn't I need to define the proc separately for every item i want to be shrapnel then ? Isn't that "dirtier" than just adding these three variables? Will initialising 3 extra variables here take up a bunch of extra processing power since they'll be initialised for all items?